connectors/source_linkedin_ads/models/fivetran_converter/account_history.sql

```diff
@@ -0,0 +1,52 @@
+{% if target.type == "snowflake" %}
+
+    with tmp as (
```

Comment:
See you must create here what [this](https://github.com/fivetran/dbt_linkedin_source/blob/f953ac9f158990a525022da732237b6f2c14cae8/models/stg_linkedin_ads__account_history.sql#L27-L39) is consuming.